### ISSUE TITLE
openai_gym_async failed to start

### ISSUE THREAD

**Issue by lynic** [reinforceio/tensorforce#440]
Env:
tensorflow-cpu 1.8.0
tensorforce 0.4.2

Run command:

```
python examples/openai_gym_async.py Pong-ram-v0 -a examples/configs/vpg.json -n examples/configs/mlp2_network.json -e 50000 -m 2000 -W 3 -M child
```

check logs, got following:

```
Traceback (most recent call last):
  File "/Users/ethanlynnl/Documents/source/elynn/tensorforce/examples/openai_gym_async.py", line 243, in <module>
    main()
  File "/Users/ethanlynnl/Documents/source/elynn/tensorforce/examples/openai_gym_async.py", line 200, in main
    network=network
  File "/usr/local/anaconda3/envs/stockforce/lib/python3.5/site-packages/tensorforce/agents/agent.py", line 283, in from_spec
    kwargs=kwargs
  File "/usr/local/anaconda3/envs/stockforce/lib/python3.5/site-packages/tensorforce/util.py", line 192, in get_object
    return obj(*args, **kwargs)
  File "/usr/local/anaconda3/envs/stockforce/lib/python3.5/site-packages/tensorforce/agents/vpg_agent.py", line 136, in __init__
    entropy_regularization=entropy_regularization
  File "/usr/local/anaconda3/envs/stockforce/lib/python3.5/site-packages/tensorforce/agents/learning_agent.py", line 141, in __init__
    batching_capacity=batching_capacity
  File "/usr/local/anaconda3/envs/stockforce/lib/python3.5/site-packages/tensorforce/agents/agent.py", line 80, in __init__
    self.model = self.initialize_model()
  File "/usr/local/anaconda3/envs/stockforce/lib/python3.5/site-packages/tensorforce/agents/vpg_agent.py", line 163, in initialize_model
    gae_lambda=self.gae_lambda
  File "/usr/local/anaconda3/envs/stockforce/lib/python3.5/site-packages/tensorforce/models/pg_model.py", line 98, in __init__
    requires_deterministic=False
  File "/usr/local/anaconda3/envs/stockforce/lib/python3.5/site-packages/tensorforce/models/distribution_model.py", line 90, in __init__
    discount=discount
  File "/usr/local/anaconda3/envs/stockforce/lib/python3.5/site-packages/tensorforce/models/memory_model.py", line 114, in __init__
    reward_preprocessing=reward_preprocessing
  File "/usr/local/anaconda3/envs/stockforce/lib/python3.5/site-packages/tensorforce/models/model.py", line 221, in __init__
    self.setup()
  File "/usr/local/anaconda3/envs/stockforce/lib/python3.5/site-packages/tensorforce/models/model.py", line 308, in setup
    self.setup_session(self.server, hooks, graph_default_context)
  File "/usr/local/anaconda3/envs/stockforce/lib/python3.5/site-packages/tensorforce/models/model.py", line 717, in setup_session
    master=server.target,
AttributeError: 'NoneType' object has no attribute 'target'
```

**Comment by lynic** [comment]
Using the master branch will not meet this error, but the training progress hang there

```
2018-07-02 17:13:07.737349: I tensorflow/core/platform/cpu_feature_guard.cc:140] Your CPU supports instructions that this TensorFlow binary was not compiled to use: AVX2 FMA                         
2018-07-02 17:13:07.738405: I tensorflow/core/distributed_runtime/rpc/grpc_channel.cc:215] Initialize GrpcChannelCache for job ps -> {0 -> 127.0.0.1:12222}    2018-07-02 17:13:07.738469: I tensorflow/core/distributed_runtime/rpc/grpc_channel.cc:215] Initialize GrpcChannelCache for job worker -> {0 -> localhost:12223, 1 -> 127.0.0.1:12224}                 
2018-07-02 17:13:07.738746: I tensorflow/core/distributed_runtime/rpc/grpc_server_lib.cc:333] Started server with target: grpc://localhost:12223               
2018-07-02 17:13:10.016128: I tensorflow/core/distributed_runtime/master_session.cc:1024] Start master session 4ef7d7b4e3fc8a8d with config:  

```

**Comment by Gor-Ren** [comment]
I was able to fix the same problem by rolling back to TF 1.5.1 CPU.

**Comment by lynic** [comment]
Thanks, rolling back to 1.5.1 did solve the problem. Still wondering why 1.8.0 couldn't train.
